geopandas/tests/test_sindex.py

```diff
@@ -504,3 +504,30 @@ def test_is_empty(self):
         non_empty = geopandas.GeoSeries([Point(0, 0)])
         tree = cls_(non_empty)
         assert not tree.is_empty
+
+    @pytest.mark.parametrize(
+        "predicate, expected_shape",
+        [
+            (None, (2, 396)),
+            ("intersects", (2, 172)),
+            ("within", (2, 172)),
+            ("contains", (2, 0)),
+            ("overlaps", (2, 0)),
+            ("crosses", (2, 0)),
+            ("touches", (2, 0)),
+        ],
+    )
+    def test_integration_natural_earth(self, predicate, expected_shape):
+        """Tests output sizes for the naturalearth datasets."""
+        world = read_file(datasets.get_path("naturalearth_lowres"))
+        capitals = read_file(datasets.get_path("naturalearth_cities"))
+        # Reproject to Mercator (after dropping Antartica)
```

Comment:
Is reprojection necessary?  I would assume that geographic coordinates should work fine for this test.

Reply:
I removed the reprojection and tested, all working. It is nicer to have less things being tested, so I think this is a good idea.